### PR TITLE
Broken link fixes and other issues

### PR DESCRIPTION
**Pull request by mairaw**
I went through the warnings in our build logs to fix issues with broken links, images, etc. Fixed other minor things I saw along the way too like notes styles, header styles, typos, on the files impacted by the broken link fixes.

/cc @cartermp @BillWagner

### CODE REVIEW

File: docs/core/porting/libraries.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Porting is a task that may take time, especially if you have a large codebase.
 
 This article assumes you are using Visual Studio 2015 or later on Windows.  The bits required for building .NET Core code are not available on previous versions of Visual Studio.
 
-This article also assumes that you have understood the [overview of the porting process](overview.md) and that you have resolved any issues with [third-party dependencies](third-party-deps.md).
+This article also assumes that you have understood the [overview of the porting process](index.md) and that you have resolved any issues with [third-party dependencies](third-party-deps.md).
```
Comment (rpetrusha) [inline docs/core/porting/libraries.md]: nit: I'd make this "assumes that you understand"

File: docs/core/porting/libraries.md [hunk 2]
```diff
@@ -49,19 +49,19 @@ A key thing to understand is that **a project targeting a lower version cannot r
...
 
 ### App Domains
 
-AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new [AssemblyLoadContext](http://dotnet.github.io/api/System.Runtime.Loader.AssemblyLoadContext.html) class.
+AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new  @System.Runtime.Loader.AssemblyLoadContext class.
 
 ### Remoting
 
-For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as [Pipes](https://.docs.microsoft.com/en-us/dotnet/core/api/system.io.pipes) or [Memory Mapped Files](hhttps://docs.microsoft.com/en-us/dotnet/core/api/system.io.memorymappedfiles.memorymappedfile#System_IO_MemoryMappedFiles_MemoryMappedFile).
+For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as @System.IO.Pipes or @System.IO.MemoryMappedFiles.MemoryMappedFile.
```
Comment (rpetrusha) [inline docs/core/porting/libraries.md]: Again a nit: The link text in the first case is clearer (pipes vs. the namespace name, "memory mapped files" versus "MemoryMappedFile"). Is there any way to use the correct syntax with text not formed from the link?

File: docs/core/porting/libraries.md (hunk 1 above)
Comment (mairaw) [inline docs/core/porting/libraries.md]: I prefer present tense too. Didn't look at the overall sentence when I fixed the link. I'll fix that.

File: docs/core/porting/libraries.md (hunk 2 above)
Comment (mairaw) [inline docs/core/porting/libraries.md]: I can roll back the change for the second one and keep as an absolute URL (just fixing the extra we have in hhttp). There's no way to format the API name like we can't format for msdn.

**Comment by rpetrusha** [comment]
I had two minor comments, which you can feel free to ignore. Otherwise, LGTM.

File: docs/core/porting/libraries.md (hunk 2 above)
Comment (rpetrusha) [inline docs/core/porting/libraries.md]: That sounds good.

**Comment by mairaw** [comment]
@cartermp, I've just added the porting libraries topic to the TOC. Can you confirm that looks good?

**Comment by cartermp** [comment]
LGTM

File: docs/csharp/delegates-strongly-typed.md [hunk 3]
```diff
@@ -67,14 +67,14 @@ The `out` modifier on the result generic type argument is covered in the
 article on covariance.
 
 There are variations of the `Func` delegate with up to
-[16 input arguments](../../core/api/system.func-17#System_Func_17).
+16 input arguments such as @System.Func%6017 .
```
Comment (richlander) [inline docs/csharp/delegates-strongly-typed.md]: That's another funky format. Can we get that one changed to match our other generic format?

Comment (mairaw) [inline docs/csharp/delegates-strongly-typed.md]: I'll open a feature request for this. The value is System.Func`17. But if I use that, then GitHub thinks it's the beginning of code.

File: docs/core/porting/libraries.md (hunk 2 above)
Comment (mairaw) [inline docs/core/porting/libraries.md]: Actually, I sent an email to @chenkennt and he told me I can actually use the @symbol as a link target. I'll give it a try tomorrow.

**Pull request merged**